src/content/guides/build-performance.md

```diff
@@ -109,6 +109,10 @@ Enable persistent caching with the `cache-loader`. Clear cache directory on `"po
 
 Profile them to not introduce a performance problem here.
 
+### Progress plugin
+
+If you already have a fast build time `progress-plugin` can be excessive. If you have long build times consider removing `progress-plugin` where the build happens to be remotely, for example your server or CI.
```

Comment:
Is it just me or does it miss comma?

```suggestion
If you already have a fast build time, `progress-plugin` can be excessive. If you have long build times, consider removing `progress-plugin` where the build happens to be remotely, for example your server or CI.
```

Reply:
Since I am not a native english speaker I am not too sure. But it makes sense.

Should I commit your suggestion?

Reply:
I'd say yes, lets get another pair of eyes at it afterwards